kornia/augmentation/container/ops.py

```diff
@@ -260,6 +266,69 @@ def transform(cls, input: Tensor, module: Module, param: ParamItem, extra_args:
             return MaskSequentialOps.transform(input, module=module.op, param=param, extra_args=extra_args)
         return input
 
+    @classmethod
+    def transform_list(
+        cls, input: List[Tensor], module: Module, param: ParamItem, extra_args: Dict[str, Any] = {}
+    ) -> List[Tensor]:
+        """Apply a transformation with respect to the parameters.
+
+        Args:
+            input: list of input tensors.
+            module: any torch Module but only kornia augmentation modules will count
+                to apply transformations.
+            param: the corresponding parameters to the module.
+        """
+        if isinstance(module, (K.GeometricAugmentationBase2D,)):
+            tfm_input = []
+            params = cls.get_instance_module_param(param)
+            for inp in input:
+                inp = module.transform_tensor(inp)
+                tfm_inp = module.transform_masks(
+                    inp, params=params, flags=module.flags, transform=module.transform_matrix, **extra_args
+                )
+                tfm_input.append(tfm_inp)
+            input = tfm_input
+
+        elif isinstance(module, (K.GeometricAugmentationBase3D,)):
+            raise NotImplementedError(
+                "The support for 3d mask operations are not yet supported. You are welcome to file a PR in our repo."
+            )
+
+        elif isinstance(module, (_AugmentationBase)):
+            tfm_input = []
+            params = cls.get_instance_module_param(param)
+            for inp in input:
+                inp = module.transform_tensor(inp)
+                tfm_inp = module.transform_masks(inp, params=params, flags=module.flags, **extra_args)
+                tfm_input.append(tfm_inp)
+            input = tfm_input
+
+        elif isinstance(module, K.ImageSequential) and not module.is_intensity_only():
+            tfm_input = []
+            seq_params = cls.get_sequential_module_param(param)
+            for inp in input:
+                tfm_inp = module.transform_masks(inp, params=seq_params, extra_args=extra_args)
+                tfm_input.append(tfm_inp)
+                input = tfm_input
+
+        elif isinstance(module, K.container.ImageSequentialBase):
+            tfm_input = []
+            seq_params = cls.get_sequential_module_param(param)
+            for inp in input:
+                tfm_inp = module.transform_masks(inp, params=seq_params, extra_args=extra_args)
+                tfm_input.append(tfm_inp)
+            input = tfm_input
```

Comment:
```suggestion
        elif isinstance(module, K.ImageSequential) and not module.is_intensity_only():
            tfm_input = []
            seq_params = cls.get_sequential_module_param(param)
            for inp in input:
                tfm_inp = module.transform_masks(inp, params=seq_params, extra_args=extra_args)
                tfm_input.append(tfm_inp)
                input = tfm_input

        elif isinstance(module, K.container.ImageSequentialBase):
            tfm_input = []
            seq_params = cls.get_sequential_module_param(param)
            for inp in input:
                tfm_inp = module.transform_masks(inp, params=seq_params, extra_args=extra_args)
                tfm_input.append(tfm_inp)
            input = tfm_input
```

@shijianjian any clue why this is falling just when running as `float64`? 🤔

Reply:
weird, I checked the code, and nowhere mentioned float32 only.

Reply:
yep, I don't understand why it's crashing the shape of the data with fp64. Maybe, because randomness isn't running when we don't put fp64. 🤔